{
  "url": "https://api.github.com/repos/shap/shap/issues/2911",
  "repository_url": "https://api.github.com/repos/shap/shap",
  "labels_url": "https://api.github.com/repos/shap/shap/issues/2911/labels{/name}",
  "comments_url": "https://api.github.com/repos/shap/shap/issues/2911/comments",
  "events_url": "https://api.github.com/repos/shap/shap/issues/2911/events",
  "html_url": "https://github.com/shap/shap/issues/2911",
  "id": 1693949766,
  "node_id": "I_kwDOBHDcK85k959G",
  "number": 2911,
  "title": "Shap not compatible with numpy>=1.24.0",
  "user": {
    "login": "vincentl-met",
    "id": 59954348,
    "node_id": "MDQ6VXNlcjU5OTU0MzQ4",
    "avatar_url": "https://avatars.githubusercontent.com/u/59954348?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/vincentl-met",
    "html_url": "https://github.com/vincentl-met",
    "followers_url": "https://api.github.com/users/vincentl-met/followers",
    "following_url": "https://api.github.com/users/vincentl-met/following{/other_user}",
    "gists_url": "https://api.github.com/users/vincentl-met/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/vincentl-met/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/vincentl-met/subscriptions",
    "organizations_url": "https://api.github.com/users/vincentl-met/orgs",
    "repos_url": "https://api.github.com/users/vincentl-met/repos",
    "events_url": "https://api.github.com/users/vincentl-met/events{/privacy}",
    "received_events_url": "https://api.github.com/users/vincentl-met/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 5,
  "created_at": "2023-05-03T12:07:59Z",
  "updated_at": "2023-06-03T02:06:11Z",
  "closed_at": "2023-06-03T02:06:11Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "Since 1.24.0 numpy removed `np.bool` which will lead to various errors using `shap` repository.\r\nOn my case I fixed `numpy` version to 1.23.0 to make it work.",
  "closed_by": {
    "login": "dsgibbons",
    "id": 82195909,
    "node_id": "MDQ6VXNlcjgyMTk1OTA5",
    "avatar_url": "https://avatars.githubusercontent.com/u/82195909?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/dsgibbons",
    "html_url": "https://github.com/dsgibbons",
    "followers_url": "https://api.github.com/users/dsgibbons/followers",
    "following_url": "https://api.github.com/users/dsgibbons/following{/other_user}",
    "gists_url": "https://api.github.com/users/dsgibbons/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/dsgibbons/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/dsgibbons/subscriptions",
    "organizations_url": "https://api.github.com/users/dsgibbons/orgs",
    "repos_url": "https://api.github.com/users/dsgibbons/repos",
    "events_url": "https://api.github.com/users/dsgibbons/events{/privacy}",
    "received_events_url": "https://api.github.com/users/dsgibbons/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/shap/shap/issues/2911/reactions",
    "total_count": 23,
    "+1": 23,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/shap/shap/issues/2911/timeline",
  "performed_via_github_app": null,
  "state_reason": "completed"
}
